README.md

```diff
@@ -83,6 +83,17 @@ Since each time the script is run the list of input files is shuffled, you don't
 workers will be processing the same video.
 On a rare occasion when the collision happens, the script will rewrite previously extracted features.
 
+
+## Input
+The inputs are paths to video files that are readable by OpenCV VideoReader.
+Paths can be passed as a list of paths or as a text file formatted with a single path per line.
+
+
+## Output
+Output is defined by the `on_extraction` argumenty; by default it prints the featrues to the command line.
```

Comment:
"argumenty" and "featrues" -- could you patch these? otherwise, no worries, I will do it myself.